splunk_add_on_ucc_framework/main.py

```diff
@@ -103,6 +103,13 @@ def main(argv: Optional[Sequence[str]] = None) -> int:
         help="Python binary name to use to install requirements",
         default="python3",
     )
+    build_parser.add_argument(
+        "-v",
+        "--verbose",
+        action="store_true",
+        default=False,
+        help="Show files install result",
```

Comment:
I'm not sure which description would fit better as it shows each file being installed